### PR TITLE
Removed white space to fix indentation issue for Beta commands.

### PR DESCRIPTION
**Pull request by SteveMutungi254**
Scope - Beta commands.

To fix [issue 1106](https://github.com/microsoftgraph/entra-powershell/issues/1106).

### CODE REVIEW

**Review by emmanuel-karanja — DISMISSED** [review]
LGTM

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md [hunk 1]
```diff
@@ -27,79 +27,79 @@ Connect to Microsoft Entra ID with an authenticated account.
 
 ```powershell
 Connect-Entra 
-[[-Scopes] <String[]>] 
-[[-ClientId] <String>] 
+[[-Scopes] <String[]>]
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md:30]: Indent parameters

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md [hunk 2]
```diff
@@ -27,79 +27,79 @@ Connect to Microsoft Entra ID with an authenticated account.
...
 ```
 
 ### AppCertificateParameterSet
 
 ```powershell
-Connect-Entra 
-[-ClientId] <String> 
-[[-CertificateSubjectName] <String>] 
+Connect-Entra
+[-ClientId] <String>
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md:45]: Indent parameters

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md [hunk 3]
```diff
@@ -27,79 +27,79 @@ Connect to Microsoft Entra ID with an authenticated account.
...
 ```
 
 ### IdentityParameterSet
 
 ```powershell
-Connect-Entra 
-[[-ClientId] <String>] 
-[-ContextScope <ContextScope>] 
+Connect-Entra
+[[-ClientId] <String>]
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md:61]: Indent parameters

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md [hunk 4]
```diff
@@ -27,79 +27,79 @@ Connect to Microsoft Entra ID with an authenticated account.
...
 ```
 
 ### AppSecretCredentialParameterSet
 
 ```powershell
-Connect-Entra 
-[-ClientSecretCredential <PSCredential>] 
-[-TenantId <String>] 
+Connect-Entra
+[-ClientSecretCredential <PSCredential>]
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md:74]: Indent parameters

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md [hunk 5]
```diff
@@ -27,79 +27,79 @@ Connect to Microsoft Entra ID with an authenticated account.
...
 
 ### AccessTokenParameterSet
 
 ```powershell
-Connect-Entra 
-[-AccessToken] <SecureString> 
-[-Environment <String>] 
-[-ClientTimeout <Double>] 
+Connect-Entra
+[-AccessToken] <SecureString>
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md:87]: Indent parameters

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md [hunk 6]
```diff
@@ -27,79 +27,79 @@ Connect to Microsoft Entra ID with an authenticated account.
...
 ```
 
 ### EnvironmentVariableParameterSet
 
 ```powershell
-Connect-Entra 
-[-ContextScope <ContextScope>] 
-[-Environment <String>] 
+Connect-Entra
+[-ContextScope <ContextScope>]
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md:98]: Indent parameters

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md (hunk 5 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Connect-Entra.md:87]: Addressed.

**Review by KenitoInc — APPROVED** [review]

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**